### ISSUE TITLE
Insert state data Sensor

### ISSUE THREAD

**Issue by innopus** [DeepBlueCLtd/tac_store#19]
Do we assume the sensor already exists in the sensor table or do we insert it new?

**Comment by IanMayo** [comment]
No, we don't assume the sensor exists.  We check for it, and only generate the row if we need to.
